Segunda_parte/ejercicio4.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+#include <string.h>
+
+/*
+Escribir un programa que cuente el numero de palabras en un string
+*/
+
+int main() {
+	char str1[100];
+	int num = 0, i;
+	
+	printf("Ingrese las palabras: ");
+	fgets(str1, sizeof(str1), stdin);
+	
+	for (i = 0; i <= strlen(str1); i++){
+		if(str1[i] == 32){
+			num ++;
+		}
+		if(str1[i] == '\n'){
+			num ++;
+		}
```

Comment:
esto puede reducirse a:

```suggestion
		if(str1[i] == 32 || str1[i] == '\n'){
			num ++;
		}```